jax/BUILD

```diff
@@ -43,7 +43,10 @@ pytype_library(
         ],
     ),
     srcs_version = "PY3",
-    deps = ["@org_tensorflow//tensorflow/compiler/xla/python:xla_client"],
+    deps = [
+        "//third_party/py/toolz",
```

Comment:
It seems odd that a toolz dependence would need to be added here, whereas no other jax Python dependencies need to be. Are you sure this is needed?